README.md

```diff
@@ -35,3 +35,9 @@ used.
 There's no trick --- just do the same transformations after renaming, rather
 than after typechecking. We realized this plugin is useful outside of
 polysemy, so it's been released as a standalone package.
+
+## Usage
+
+Add `-fplugin=Loopbreaker` into your `package.yaml` or specific module and
```

Comment:
```suggestion
Add `-fplugin=Loopbreaker` into your `package.yaml` or specific module. This will cause the plugin to generate explicit loopbreakers for any self-recursive functions marked as `INLINE`, which can dramatically improve their performance.
```

Most people have no idea what a loopbreaker is, or why they should care :)